### PR TITLE
Slide/Click button at confirm view

### PR DESCRIPTION
**Pull request by JDonadio**
Fixes https://github.com/bitpay/copay/issues/5436

* Set "Slide/Click to" instead of just "Accept"
* Fix translations
* Fix display slider if:
    - No wallet
    - No matching wallet
    - Insufficient funds

### CODE REVIEW

**Comment by Kirvx** [comment]
:point_right: https://github.com/JDonadio/copay/pull/57 👈 
---------:fr: :ok_hand:--------

File: src/js/controllers/modals/txpDetails.js [hunk 1]
```diff
@@ -19,15 +19,29 @@ angular.module('copayApp.controllers').controller('txpDetailsController', functi
...
     initActionList();
     checkPaypro();
-  }
+    applyButtonText();
+  };
+
+  function applyButtonText() {
+    $scope.buttonText = $scope.isCordova ? gettextCatalog.getString('Slide') + ' ' : gettextCatalog.getString('Click') + ' ';
+
+    var lastSigner = lodash.filter($scope.tx.actions, {
```
Comment (gabrielbazan7) [inline src/js/controllers/modals/txpDetails.js]: 👏

File: src/js/controllers/confirm.js [hunk 2]
```diff
@@ -268,6 +282,8 @@ angular.module('copayApp.controllers').controller('confirmController', function(
       $scope.getSendMaxInfo();
     } else
       setWallet(wallet);
+
+    applyButtonText(wallet.credentials.n > 1);
```
Comment (cmgustavo) [inline src/js/controllers/confirm.js]: `n` is number of copayers. `m` is the number of required signature.

I think that it should be `m`.

Comment (JDonadio) [inline src/js/controllers/confirm.js]: Done, good catch!

**Pull request merged**